examples/mario/play_mario.html

```diff
@@ -134,7 +132,7 @@
       <button  id="trackbutton" class="bx--btn bx--btn--secondary" type="button" pys-onClick="toggle_video">
         Start Video
       </button>
-      <div id="update-note" py-mount class="updatenote mt10">loading model ..</div>
+      <div id="update-note" py-mount class="updatenote mt10" style="display: none;"">loading model ..</div>
```

Comment:
I think this change is correct. What's the intent here?

Reply:
Because when I was testing I saw that sometimes the loader crashes so no need to see this message if the loader is not loading, I will work on it later but for now hiding it will be the fast solution

Reply:
I will work on the loading errors when I have some time